### ISSUE TITLE
Fail on terminate namespace

### ISSUE THREAD

**Issue by jparrill** [mvazquezc/termin8#1]
Trying to delete a namespace the output shows something like this:

```
termin8 run -n open-cluster-management-agent-addon

✗ Couldn't get stuck resources in namespace open-cluster-management-agent-addon
Error: unable to retrieve the complete list of server APIs: admission.agentinstall.openshift.io/v1: stale GroupVersion discovery: admission.agentinstall.openshift.io/v1, clusterview.open-cluster-management.io/v1: stale GroupVersion discovery: clusterview.open-cluster-management.io/v1, clusterview.open-cluster-management.io/v1alpha1: stale GroupVersion discovery: clusterview.open-cluster-management.io/v1alpha1, proxy.open-cluster-management.io/v1beta1: stale GroupVersion discovery: proxy.open-cluster-management.io/v1beta1
```

**Comment by jparrill** [comment]
Being addressed in #2

**Comment by mvazquezc** [comment]
Thanks for submitting the fix!
